staging/src/kubevirt.io/client-go/kubecli/handler.go

```diff
@@ -262,6 +268,42 @@ func (v *virtHandlerConn) Get(url string) (string, error) {
 	return response, nil
 }
 
+func (v *virtHandlerConn) GetStream(url string, tlsConfig *tls.Config, response *restful.Response) error {
```

Comment:
I don't think we need a plain function for the `GetStream`. Basically, we can use the `Get` function. It will not work because of the header, but we can edit it with something like:
```suggestion
func (v *virtHandlerConn) Get(url string, headers map[string]string) (string, error) {
	req, err := http.NewRequest(http.MethodGet, url, nil)
	if err != nil {
		return "", err
	}

	for key, value := range headers {
		req.Header.Add(key, value)
	}
	response, err := v.doRequest(req)
	if err != nil {
		return "", err
	}

	return response, nil
}
```
And pass the headers from the caller.